app/models/event-sub-topic.js

```diff
@@ -6,6 +6,7 @@ export default ModelBase.extend({
   name : attr('string'),
   slug : attr('string'),
 
-  topic  : belongsTo('event-topic'),
-  events : hasMany('event')
+  topic       : belongsTo('event-topic'),
+  events      : hasMany('event'),
+  placeholder : belongsTo('custom-placeholder')
```

Comment:
won't it expect customPlaceholder otherwise? If I am wrong, then please explain to me why event-topic, sub-topic have been serialized . 😅